tests/core/arch-arm64/arm64stalker.c

```diff
@@ -0,0 +1,652 @@
+/*
+ * Copyright (C) 2009-2014 Ole André Vadla Ravnås <ole.andre.ravnas@tillitech.com>
+ * Copyright (C) 2010-2013 Karl Trygve Kalleberg <karltk@boblycat.org>
+ *
+ * Licence: wxWindows Library Licence, Version 3.1
+ */
+
+#include "arm64stalker-fixture.c"
```

Comment:
Yes, because in the other case the fully-qualified class name is `Gum.Arm64Relocator`, but here it is `Gum.Stalker`, and this is test-code for its arm64 backend code.